packages/rag/main.py

```
@@ -0,0 +1,139 @@
+            "context_precision": None
+        }
+        
+        # DataFrame에서 점수 추출 (NaN 처리 포함)
+        if idx < len(df):
+            row = df.iloc[idx]
+            
+            # 안전한 float 변환 함수
+            def safe_float(val):
+                try:
+                    return float(val) if pd.notnull(val) else None
+                except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)